### PR TITLE
REF use conda like we had it

### PR DESCRIPTION
**Pull request by beckermr**
This PR moves the compilers back to conda like it was before.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -5,132 +5,124 @@ on:
...
+      - uses: actions/checkout@v2
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.py }}
+          channels: conda-forge,defaults
+          channel-priority: strict
+          show-channel-urls: true
+
+      - name: remove homebrew
```
Comment (beckermr) [inline .github/workflows/ci.yml:40]:
```suggestion
      - name: remove homebrew
        if: matrix.os == 'macos-latest'
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -5,132 +5,124 @@ on:
...
+            numpy nose coveralls flake8 pyyaml gsl fftw cmake swig scipy \
+            compilers pkg-config setuptools_scm pytest pandas pytest-cov \
+            cython "camb>=1" isitgr traitlets fast-pt
+
+          if [[ ${MATRIX_OS} == "macos-latest" ]]; then
+            conda install llvm-openmp
+            echo "DYLD_FALLBACK_LIBRARY_PATH=${CONDA_PREFIX}/lib" >> $GITHUB_ENV
+            SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
+            echo "SDKROOT: ${SDKROOT}"
+            # echo "SDKROOT=${SDKROOT}" >> $GITHUB_ENV
```
Comment (beckermr) [inline .github/workflows/ci.yml:60]:
```suggestion
            echo "SDKROOT=${SDKROOT}" >> $GITHUB_ENV
```

File: ci_scripts/install_class_linux.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+git clone https://github.com/lesgourg/class_public.git
+cd class_public
+git checkout v2.7.2
+make
```
Comment (beckermr) [inline ci_scripts/install_class_linux.sh:6]:
```suggestion
make -j4
```

File: ci_scripts/install_class_osx.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+git clone --depth=1000 https://github.com/lesgourg/class_public.git
+cd class_public
+git checkout v2.7.2
+
+sed -i.bak -e 's/^CC/#CC/g' Makefile
+sed -i.bak -e 's/^OPTFLAG =/OPTFLAG = ${CFLAGS} ${LDFLAGS}/g' Makefile
+sed -i.bak -e 's/^#CCFLAG +=/CCFLAG +=/g' Makefile
+sed -i.bak -e 's/^#CCFLAG =/CCFLAG =/g' Makefile
+
+make
```
Comment (beckermr) [inline ci_scripts/install_class_osx.sh:15]:
```suggestion
make -j4
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -5,132 +5,125 @@ on:
       - main
       - master
       - releases/*
-  pull_request:
-    branches:
-      - main
-      - master
-      - releases/*
+  pull_request: null
```
Comment (damonge) [inline .github/workflows/ci.yml:8]: What does this do?

File: .github/workflows/ci.yml [hunk 6]
```diff
@@ -5,132 +5,125 @@ on:
...
-
-        # Check macos too
-        include:
-          - os: macos-latest
-            py: 3.7
+        os:
+          - macos-latest
+          - ubuntu-latest
+        py:
+          - 3.8
```
Comment (damonge) [inline .github/workflows/ci.yml:23]: should we check 3.9 too?

Comment (damonge) [reply .github/workflows/ci.yml:23]: (and 3.6?)

**Review by damonge — APPROVED** [review]
Looks good! Just a couple of questions, but happy to merge. Thanks!

File: .github/workflows/ci.yml (hunk 6 above)
Comment (beckermr) [reply .github/workflows/ci.yml:23]: No I don't think so.

**Pull request merged**